plugins/API/ProcessedReport.php

```diff
@@ -715,9 +715,17 @@ private function aggregateReportTotalValues($simpleDataTable, $totals)
         $simpleTotals = $this->hideShowMetrics($metadataTotals);
 
         foreach ($simpleTotals as $metric => $value) {
+            if (0 === strpos($metric, 'avg_') || '_rate' === substr($metric, -5) ||
+                0 === strpos($metric, 'min_') || 0 === strpos($metric, 'max_')) {
```

Comment:
@sgiehl in general we do want min/max values to be considered here. `$row->sumRow` should notice whether it's min/max etc. and if not we could make the `sumRow` logic understands what to do. You actually want to have the max value of all of the rows if there is such a metric for example so you can apply processed metrics etc. if / when needed and allows for quite useful features maybe.

Reply:
ok. will fix the calculation for min/max values then instead. Wasn't sure what the feature is actually used for besides the totals row (that shouldn't be displayed for such metrics)